app/models/concerns/manage_user_roles.rb

```diff
@@ -0,0 +1,44 @@
+module ManageUserRoles
```

Comment:
@swatijadhav it would be good to have some unit level specs for this module. I know a lot of integration specs are covered in the users_controller_spec but here is a good place to have `specs/models/concerns/manage_user_roles_spec.rb` to cover a more unit style approach.